File: lg/php/caesar.php
<center>
<form method="post">
plaintext:<input type="text" name="plaintext"><br>
<input type="submit">
</form>
</center>
<?php
echo "ciphertext: \r\n";
$i = 0;
$str = $_POST["plaintext"];
for($i = 0; $i<= strlen($str); $i++)
{
	$tmp = ord($str[$i]);
	if($tmp >= 97 && $tmp <= 122){
		$tmp = 97 + ($tmp - 97 + 5) % 26;
		echo chr($tmp);
	}
	else if($tmp >= 65 && $tmp <= 90){
		$tmp = 65 + ($tmp - 65 + 5) % 26;
		echo chr($tmp);
	}
	
}
?>

File: lg/ruby/caesar.rb
#!/usr/bin/ruby
puts "input plaintext";
plaintext = gets;
i = 0;
puts "ciphertext is :"
while i < plaintext.length do
	ch = plaintext[i].ord;
	if ch >= 97 and ch <= 122 then
		print (97 + (ch - 97 + 5) % 26).chr;
	elsif ch >= 65 and ch <= 90 then
		print (65 + (ch - 65 + 5) % 26).chr;
	else 
		print ch.chr;
	end
	i = i + 1
end
puts ""
